app/questionnaire/placeholder_transforms.py

```diff
@@ -125,6 +125,36 @@ def parse_date(date):
                 date, PlaceholderTransforms.input_date_format_month_year_only
             ).replace(tzinfo=tzutc())
 
+    @staticmethod
+    def add(lhs, rhs):
+        if isinstance(lhs, str):
+            return int(lhs) + rhs
+        return lhs + rhs
+
+    @staticmethod
+    def format_ordinal(number_to_format, determiner=None):
+        number_to_format_str = str(number_to_format)
+
+        if number_to_format_str.endswith(("11", "12", "13")):
+            suffix = "th"
+        elif number_to_format_str.endswith("1"):
+            suffix = "st"
+        elif number_to_format_str.endswith("2"):
+            suffix = "nd"
+        elif number_to_format_str.endswith("3"):
+            suffix = "rd"
+        else:
+            suffix = "th"
+        if determiner == "a_or_an":
```

Comment:
I probably wouldn't do both in the same method, one method to do the prefix one for the suffix, making it more reusable, we might use a_or_an somewhere else

Reply:
"determiner" is part of the same transform as proposed on [this card](https://trello.com/c/PttjF0kg/3537-b-ordinals-eg-third-in-schema-text-m). Would have to be a completely different transform?